web/lib/queries/common/pageContentFields.ts

```diff
@@ -38,6 +38,22 @@ const pageContentFields = /* groq */ `
       ${downloadableImageFields},
     },
   },
+
+  _type == "textTeaser" => {
```

Comment:
extract into its own file as we did for the latest added components - so we have a readable/scalable groq in the future.